src/Autofac/Autofac.csproj

```diff
@@ -71,10 +71,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3">
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0">
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0">
```

Comment:
I _think_ we don't actually need to reference the `NetAnalyzers` package after .NET 6 - it comes "built in" with the compilation.